recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
 
   run:
     - python
-    - aiobotocore ==2.4.0
+    - aiobotocore ~=2.4.0
```

Comment:
I don't think conda understands ~= (https://peps.python.org/pep-0440/#compatible-release)
Instead you could set: `- aiobotocore >=2.4.0,==2.4.*`

Reply:
Good to know! Thank you.